src/core_plugins/kibana/public/assets/play-circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8">
+  <path d="M4 0c-2.21 0-4 1.79-4 4s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm-1 2l3 2-3 2v-4z" />
+</svg>
```

Comment:
It seems to have more in common with the other icons in `src/core_plugins/kibana/public/assets`. Do you mean to say that the kibana core_plugin is optional, so this icon could possibly be excluded from someone's deployment?